spec/SPEC-annex-C-source-embedding.md

```diff
@@ -0,0 +1,95 @@
+## Annex C
+
+Annex C contains a method of embedding an Input Manifest Identifier into source code files files.
+
+### Embedded Input Manifest Identifier
+
+Most source code files are hand coded by humans.  Some however are generated from other input(s) by a build tool.
+
+A build tool outputing a source code file may embed the Input Manifest Identifier for the output source code file into
+the output source code file by adding a comment line containing a string of the form:
+
+``` OmniBOR-Input-Manifest-ID: [ ${comma separated list of Input Manifest Identifier URIs} ]
```

Comment:
GitHub doesn't appear to like these inline triple-backticked code blocks. If the intent is for a codeblock to be inline, it should use single-backticks. If it the intent is for it to be a non-inline codeblock, then the triple backticks should each appear on their own lines.

Also, when we say "comma separated" we should clarify the spacing rules (if any).

Reply:
> Also, when we say "comma separated" we should clarify the spacing rules (if any).

Proposed fix in 09a479ae8ff6b335cfe6412aa6446f92d29e319e